pkg/controller/cleanup/cleanup.go

```diff
@@ -110,7 +110,8 @@ func (c *Controller) HandleCleanup() http.Handler {
 			}
 		}()
 
-		// Verification codes
+		// Verification codes - purge codes from database entirerly.
+		// Their code/long_code hmac values will have been set to "".
```

Comment:
This comment is a little confusing since the value will be NULL, not "".

Reply:
I was inconsistent.
Since the type in gorm isn't a *string, it won't allow NULL to be set, even w/ the update statement.
These get set to empty string.
Change to be consistent.